server/router/activeride.ts

```diff
@@ -8,11 +8,23 @@ const router = express.Router();
 AWS.config.update(config);
 const docClient = new AWS.DynamoDB.DocumentClient();
 
+type ActiveRide = {
+  id: string,
+  startLocation: string,
+  endLocation: string,
+  startTime: string,
+  endTime: string,
+  isScheduled: boolean,
+  riderID: string[],
+  driverID: string[] | null,
```

Comment:
until i think of a better solution, for now the driver IDs will be ordered depending on the days listed in repeatsOn. if repeatsOn doesn't exist, the array will only have one ID